docs/qcvv/xeb_characterization_pipeline.ipynb

```diff
@@ -0,0 +1,1818 @@
+       "            + ' to learn more about interactive tables.';\n",
+       "          element.innerHTML = '';\n",
+       "          dataTable['output_type'] = 'display_data';\n",
+       "          await google.colab.output.renderOutput(dataTable, element);\n",
+       "          const docLink = document.createElement('div');\n",
+       "          docLink.innerHTML = docLinkHtml;\n",
+       "          element.appendChild(docLink);\n",
+       "        }\n",
+       "      </script>\n",
+       "    </div>\n",
+       "  </div>\n",
+       "  "
+      ],
+      "text/plain": [
+       "                                        a  layer_fid         cycle_depths  \\\n",
+       "layer_i pair_i pair                                                         \n",
+       "0       0      ((4, 4), (5, 4))  0.985863   0.979099  [3, 23, 43, 63, 83]   \n",
+       "        1      ((5, 3), (6, 3))  0.983222   0.978931  [3, 23, 43, 63, 83]   \n",
+       "1       0      ((4, 3), (5, 3))  0.997131   0.978314  [3, 23, 43, 63, 83]   \n",
+       "2       0      ((4, 3), (4, 4))  0.988668   0.978966  [3, 23, 43, 63, 83]   \n",
+       "3       0      ((5, 3), (5, 4))  0.982421   0.979230  [3, 23, 43, 63, 83]   \n",
+       "\n",
+       "                                                                        fidelities  \\\n",
+       "layer_i pair_i pair                                                                  \n",
+       "0       0      ((4, 4), (5, 4))  [0.9237841043826969, 0.609419685380498, 0.3978...   \n",
+       "        1      ((5, 3), (6, 3))  [0.923059933482562, 0.599137318358399, 0.39757...   \n",
+       "1       0      ((4, 3), (5, 3))  [0.9320436161769315, 0.6055788537158963, 0.387...   \n",
+       "2       0      ((4, 3), (4, 4))  [0.928302050887004, 0.6051568513305658, 0.3973...   \n",
+       "3       0      ((5, 3), (5, 4))  [0.9234175841861382, 0.6024683665375963, 0.399...   \n",
+       "\n",
+       "                                    a_std  layer_fid_std  \n",
+       "layer_i pair_i pair                                       \n",
+       "0       0      ((4, 4), (5, 4))  0.003026       0.000118  \n",
+       "        1      ((5, 3), (6, 3))  0.003956       0.000155  \n",
+       "1       0      ((4, 3), (5, 3))  0.004200       0.000166  \n",
+       "2       0      ((4, 3), (4, 4))  0.004956       0.000193  \n",
+       "3       0      ((5, 3), (5, 4))  0.007927       0.000308  "
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from cirq.experiments.xeb_fitting import fit_exponential_decays, exponential_decay\n",
+    "fidelities = fit_exponential_decays(fids)\n",
+    "fidelities.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4InvJNVJWV3H"
+   },
+   "source": [
+    "# Visualizations\n",
```

Comment:
In general, the descriptions for each section are pretty light.

Reply:
The descriptions in this tutorial are designed to be short and to-the-point, to make it easier to use the tutorial as copy-paste text snippets. However, this could definitely do with a sentence or so on why we want these visualizations and one might use them.

Reply:
Some of those sentences have been expanded!